server/entity/src/broadcasters/movement.rs

```diff
@@ -22,39 +22,40 @@ pub fn broadcast_movement(game: &mut Game, world: &mut World) {
         world.inner(),
         |(entity, (pos, prev_pos, id))| {
             let pos: Position = *pos;
-            let prev_pos: Position = prev_pos.0;
 
-            if pos == prev_pos {
-                return;
-            }
+            if let Some(prev_pos) = prev_pos.0 {
```

Comment:
Just a minor style comment—this `if let` produces some more nested indentation. I'd prefer something like

```rust
let prev_pos = match prev_pos.0 {
     Some(prev_pos) => prev_pos,
    None => return,
}
```
to reduce nesting.